src/main/scala/dal/table/userDetailTable.scala

```diff
@@ -0,0 +1,14 @@
+package dal.table
+
+import sqlest._
+
+class UserDetailTable(alias: Option[String]) extends Table(user_detail) {
+
+  val userID = column[Int]("user_ID")
+  val name = column[String]("name")
+  val salutation = column[String]("salutation")
+  val emailAddress = column[String]("email_address")
```

Comment:
We will need password and salt here for the login process. I don't think we need them in the corresponding User domain class but we'll need a method to do the login logic and that will involve looking up the db values.